test/unit/runner.spec.js

```diff
@@ -344,9 +344,13 @@ describe('Runner', function() {
   describe('.failHook(hook, err)', function() {
     it('should increment .failures', function() {
       expect(runner.failures, 'to be', 0);
-      runner.failHook(new Test('fail hook 1', noop), {});
+      var test1 = new Test('fail hook 1', noop);
+      var test2 = new Test('fail hook 2', noop);
+      suite.addTest(test1);
+      suite.addTest(test2);
+      runner.failHook(test1, {});
```

Comment:
Shouldn't second argument be an `Error`?

Reply:
It's an existing test, I did not change that part. It works fine like this.

Reply:
Does this work?
```js
runner.failHook(test1, new Error('bummer'));
```

Reply:
Same as comment below -- create `Error`.

Reply:
I was going to say it wasn't worth requesting, but @plroebuck changed my mind.

It *is* irrelevant to the test, but the test becomes more tightly coupled to the internal implementation if we don't pass an `Error`.  probably best to change it.